site/gatsby-site/cypress/fixtures/classifications/upsertCSET.json

```diff
@@ -0,0 +1,94 @@
+{
+    "data": {
```

Comment:
What is the lifecycle of the fixtures when running tests locally versus in the build system? This may need to be added to the README.

Reply:
In this case, it is just a fancy way to load a JSON file; it will always do the same thing no matter the env.